### ISSUE TITLE
Very low mean iou

### ISSUE THREAD

**Issue by anandthegreat** [Dhr11/Semantic_Segmentation#1]
Even after training for more than 3 days on 10000+ epochs, the mean iou comes out to be 3.6%. The validation loss is around 5%, then why the mean iou is so low? I am new to deep learning, please help.

![image](https://user-images.githubusercontent.com/39555945/112258589-c0a0b600-8c8c-11eb-9fe1-77e01f9ddd0e.png)

**Comment by Qiang-Feng98** [comment]
> Even after training for more than 3 days on 10000+ epochs, the mean iou comes out to be 3.6%. The validation loss is around 5%, then why the mean iou is so low? I am new to deep learning, please help.
> 
> ![image](https://user-images.githubusercontent.com/39555945/112258589-c0a0b600-8c8c-11eb-9fe1-77e01f9ddd0e.png)

I met the same problem. Have you solved it ?
